client/src/components/modals_and_popovers/bootstrap_modal_exstension.scss

```diff
@@ -36,4 +36,15 @@ $margin: 30px;
   & > svg {
     margin-right: 0.5em
   }
+}
+
+
+.modal-sm {
+  width: 300px;
+}
+.modal-lg {
+  width: 800px;
+}
+.modal-xl {
+  width: 1140px;
 }
```

Comment:
Oh, these are ours, that makes it easier to add the responsiveness. Maybe a third class `modal-responsive` the appropriate media queries build in?

Reply:
These classes are copied from bootstrap 4's modals, but I don't know if there's a case where we wouldn't want it to be responsive, so might as well just have that, no?